src/articles/dto/article-response.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { Article } from '@prisma/client';
+import { Transform } from 'class-transformer';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { capitalizeTitle } from './articles-response.dto';
+
+export class ArticleResponseDto implements Article {
+  id: number;
+  @IsString()
+  @IsNotEmpty()
+  @Transform(capitalizeTitle)
+  title: string;
+
+  @IsString()
+  @IsOptional()
+  text: string | null;
```

Comment:
You were supposed to transform the `text` to always begin with the uppercase letter